promgen/migrations/0030_audit_parent_content_type_id_audit_parent_object_id_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.11 on 2025-08-12 13:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("promgen", "0029_migrate_owners_to_admins"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="audit",
+            name="parent_content_type_id",
+            field=models.PositiveIntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name="audit",
+            name="parent_object_id",
+            field=models.PositiveIntegerField(default=0),
+        ),
+        migrations.AddIndex(
+            model_name="audit",
+            index=models.Index(
+                fields=["parent_content_type_id", "parent_object_id"],
+                name="promgen_aud_parent__4cbb33_idx",
+            ),
+        ),
+    ]
```

Comment:
We should be able to create the new columns and populate them in the same migration file.

Please apply this patch: 
[0002-fixup-Modify-the-Audit-model-to-improve-the-Audit-Lo.PATCH](https://github.com/user-attachments/files/21747719/0002-fixup-Modify-the-Audit-model-to-improve-the-Audit-Lo.PATCH)